public/app/core/components/NavBar/NavBarMenu.tsx

```diff
@@ -459,3 +459,6 @@ const getCollapsibleStyles = (theme: GrafanaTheme2) => ({
 function linkHasChildren(link: NavModelItem): link is NavModelItem & { children: NavModelItem[] } {
   return Boolean(link.children && link.children.length > 0);
 }
+function toIconName(icon: string | undefined): IconName | undefined {
+  throw new Error('Function not implemented.');
+}
```

Comment:
i don't understand this bit?

Reply:
I pressed the wrong button. its fixed now 😌 

![image](https://user-images.githubusercontent.com/46142/187905699-90873c4f-ef1c-45e7-a169-81565111ad40.png)